recipes/python/meta.yaml

```diff
@@ -0,0 +1,56 @@
+package:
+  name: python
+  version: "3.5.1"
```

Comment:
Also, could extract all of these version numbers and use jinja to make this simpler to maintain.